pkg/service/command.go

```diff
@@ -0,0 +1,21 @@
+package service
+
+import (
+	"fmt"
+	"strings"
+
+	redis "github.com/reborndb/go/redis/resp"
+)
+
+type CommandFunc func(c *conn, args [][]byte) (redis.Resp, error)
```

Comment:
conn is not exported, so it is not proper here for outer register. 

Maybe we can still use a Session interface{}, and I don't want outer to use conn directly.